source/unstyled/_hover_menu.html.erb

```diff
@@ -0,0 +1,10 @@
+<li class="nav-link hover-menu"><a href="javascript:void(0)">More</a>
```

Comment:
Have you thought about moving the `a` element to a new line? It gets lost over there…